src/onboarding/registration/OnboardingRecoveryPhrase.tsx

```diff
@@ -0,0 +1,202 @@
+      headerTitle: () => {
+        return (
+          <HeaderTitleWithSubtitle
+            title={t('protectWallet.title')}
+            subTitle={t('registrationSteps', { step, totalSteps })}
+          />
+        )
+      },
+      headerRight: () => (
+        <TopBarTextButton
+          testID="helpButton"
+          title={t('help')}
+          onPress={onPressHelp}
+          titleStyle={{ color: colors.onboardingBrownLight }}
+        />
+      ),
+    })
+  }, [navigation, step, totalSteps])
+
+  const onPressHelp = () => {
+    ValoraAnalytics.track(OnboardingEvents.protect_wallet_help)
+    setShowBottomSheet(true)
+  }
+  const onPressDismissBottomSheet = () => {
+    ValoraAnalytics.track(OnboardingEvents.protect_wallet_help_dismiss)
+    setShowBottomSheet(false)
+  }
+  const onPressCopy = () => {
+    ValoraAnalytics.track(OnboardingEvents.protect_wallet_copy_phrase)
+    Clipboard.setString(accountKey || '')
+    Logger.showMessage(t('recoveryPhrase.mnemonicCopied'))
+  }
+  const onPressContinue = () => {
+    ValoraAnalytics.track(OnboardingEvents.protect_wallet_complete)
+    goToNextOnboardingScreen({ firstScreenInCurrentStep: Screens.ProtectWallet, onboardingProps })
+  }
+
+  return (
+    <SafeAreaView style={styles.container}>
+      <ScrollView contentContainerStyle={styles.contentContainer}>
+        <Text style={styles.recoveryPhraseTitle}>{t('recoveryPhrase.title')}</Text>
+        <Text style={styles.recoveryPhraseBody}>
+          {t('recoveryPhrase.body', { mnemonicLength })}
+        </Text>
+        <BackupPhraseContainer
+          readOnlyStyle={styles.backupPhrase}
+          value={accountKey}
+          mode={BackupPhraseContainerMode.READONLY}
+          type={BackupPhraseType.BACKUP_KEY}
+          includeHeader={false}
+        />
+        <View style={styles.bottomSection}>
+          <TextButton
+            style={styles.copyButtonStyle}
+            onPress={onPressCopy}
+            testID={'protectWalletCopy'}
+          >
+            <View style={styles.copyIconStyle}>
+              <CopyIcon color={colors.onboardingGreen} />
+            </View>
+            {t('recoveryPhrase.copy')}
+          </TextButton>
+          <Button
+            onPress={onPressContinue}
+            text={t('recoveryPhrase.continue')}
+            size={BtnSizes.FULL}
+            type={BtnTypes.ONBOARDING}
+            testID={'protectWalletBottomSheetContinue'}
+          />
+        </View>
+      </ScrollView>
+
+      <BottomSheet
+        testID="OnboardingRecoveryPhraseBottomSheet"
+        isVisible={showBottomSheet}
+        onBackgroundPress={onPressDismissBottomSheet}
+      >
+        <View>
+          <Text style={styles.bottomSheetTitle}>{t('recoveryPhrase.bottomSheet.title')}</Text>
+          <Text style={styles.bottomSheetBody}>{t('recoveryPhrase.bottomSheet.body1')}</Text>
+          <Text style={styles.bottomSheetBody}>{t('recoveryPhrase.bottomSheet.body2')}</Text>
```

Comment:
these translation keys don't tell me what is being translated. I think this makes it harder for our translators too, since the base translation and the key are all they have to go off to think of how the string is used

Reply:
good callout; updated these to be a bit more descriptive.